zerver/actions/message_send.py

```diff
@@ -223,12 +225,16 @@ def get_recipient_info(
                 ),
                 user_profile_push_notifications=F("user_profile__enable_stream_push_notifications"),
                 user_profile_wildcard_mentions_notify=F("user_profile__wildcard_mentions_notify"),
+                followed_topic_email_notifications=F(
```

Comment:
We might want to call these e.g. `user_profile_followed_topic_email_notifications`, for future-proofing in case in the future we offer different "followed topic" overrides for individual streams...

But I think there's no reason to do that preemptively, given that these variables names are entirely local to this function and the longer versions would be very cumbersome.

Reply:
> But I think there's no reason to do that preemptively, given that these variables names are entirely local to this function and the longer versions would be very cumbersome.

Didn't rename in favour of the above statement.